tests/TestDocumentation.json

```diff
@@ -0,0 +1,418 @@
+[
```

Comment:
What is this file for?

Reply:
Code accidently added for another pr, removed